src/config.ts

```diff
@@ -168,7 +168,7 @@ export class Config {
   }
 
   has(key: string) {
-    return _.has(this.data, key);
+    return this.data[key];
```

Comment:
To keep the return type the same:

```suggestion
    return !!this.data[key];
```